src/components/modals/ManageProjectAddresses/AddNewAddress.tsx

```diff
@@ -39,18 +41,25 @@ export const AddNewAddress: FC<IAddNewAddress> = ({
 		formState: { errors },
 		setError,
 	} = useForm<IAddressForm>({ mode: 'onSubmit', reValidateMode: 'onSubmit' });
+	const chainType =
+		'chainType' in selectedChain ? selectedChain.chainType : undefined;
 
 	const handleAdd = async (formData: IAddressForm) => {
 		setLoading(true);
 		const { address } = formData;
-		console.log('ASD1', address);
 		try {
-			const _address = getAddress(address) as Address;
+			let _address: string;
+			if (chainType === ChainType.SOLANA) {
+				_address = address;
+			} else {
+				_address = getAddress(address);
+			}
```

Comment:
And also we can replace these lines with this
`const _address =  chainType === ChainType.SOLANA ? address : getAddress(address)`

Reply:
Done!